SoftLayer/managers/hardware.py

```diff
@@ -288,6 +335,65 @@ def place_order(self, **kwargs):
         :param boolean no_public: True if this server should only have private
                                   interfaces
         :param list extras: List of extra feature names
+
+        .. warning::
```

Comment:
This example/description is no longer true. place_order uses the simplified hardware ordering and is able to take more-or-less human readable values. See the new `slcli server create-options` and `slcli server create` commands to see how it works now.